test/libsolidity/semanticTests/constructor/constructor_function_argument.sol

```diff
@@ -0,0 +1,9 @@
+// The IR of this test used to throw
+contract D {
+    constructor(function() external returns (uint)) {
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// constructor(): 0xfdd67305928fcac8d213d1e47bfa6165cd0b87b946644cd0000000000000000 ->
```

Comment:
@bshastry Fixed the call! I got the value from `test/libsolidity/semanticTests/tryCatch/return_function.sol`

Reply:
Thanks. Just curious: What happens if you place a dummy u256 here since the actual value does not matter (unused param)?

Reply:
Yeah, I think anything should work here.